cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/guice/SourceControlModule.java

```diff
@@ -27,5 +29,6 @@ public class SourceControlModule extends AbstractModule {
   @Override
   protected void configure() {
     bind(SourceControlOperationRunner.class).toProvider(SourceControlOperationRunnerProvider.class);
+    bind(RepositoryCleanupService.class).in(Scopes.SINGLETON);
```

Comment:
Do we need the binder here?

Reply:
IMO We need the service to be a singleton to avoid multiple cleanup jobs running at the same time